gnomad_qc/v3/sample_qc/sample_qc.py

```diff
@@ -474,6 +476,8 @@ def assign_pops(
     max_mislabeled_training_samples: int = 50,  # TODO: Think about this parameter and add it to assign_population_pcs. Maybe should be a fraction? fraction per pop?
     n_pcs: int = 16,
     withhold_prop: float = None,
+    subpops: bool = False,
+    subpops_pca_scores: hl.Table = None,
```

Comment:
I think we might actually want to change this. Let's add a `high_quality` option to be passed to `ancestry_pca_scores` and a `pop` with default of None. Then `pop_pca_scores_ht = ancestry_pca_scores(include_unreleasable_samples, high_quality, pop).ht()` 

Then you don't need the check for `subpops_pca_scores` and if it doesn't exist, that will be obvious from trying to get `ancestry_pca_scores`

Then all that really changes is how `training_pop` is defined

Reply:
I was debating those options at first, but would that break what was previously done for population assignment then?